src/test/java/com/github/iveina2021/page/MainPage.java

```diff
@@ -11,8 +12,43 @@ public class MainPage {
     @FindBy(css = "a[href='/login']")
     public SelenideElement signupLoginLink;
 
+    @FindBy(css = "a[href='/delete_account']")
+    public SelenideElement deleteAccountLink;
+
+    @FindBy(xpath = "//a[.//b]")
```

Comment:
(strange selector) I can't even understand what it means

Reply:
Edited this selector